common/persistence/visibility/defs.go

```diff
@@ -74,17 +74,23 @@ func GetSecondaryVisibilityWritingModeConfig(
 	)
 }
 
-func AllowListForValidation(storeNames []string) bool {
+func AllowListForValidation(
+	storeNames []string,
+	allowList dynamicconfig.BoolPropertyFnWithNamespaceFilter,
+) dynamicconfig.BoolPropertyFnWithNamespaceFilter {
 	if len(storeNames) == 0 {
-		return false
+		return dynamicconfig.GetBoolPropertyFnFilteredByNamespace(false)
 	}
 
 	switch storeNames[0] {
 	case mysql.PluginNameV8, postgresql.PluginNameV12, postgresql.PluginNameV12PGX, sqlite.PluginName:
 		// Advanced visibility with SQL DB don't support list of values
-		return false
+		return dynamicconfig.GetBoolPropertyFnFilteredByNamespace(false)
+	case mysql.PluginName, postgresql.PluginName, postgresql.PluginNamePGX:
+		// Standard visibility with SQL DB don't support list of values, but pass validation
+		return dynamicconfig.GetBoolPropertyFnFilteredByNamespace(true)
```

Comment:
Why do we default to `true` here if standard SQL vis doesn't support lists of values?

Reply:
In the past it was possible to add search attributes with standard visibility. You'd be able to set search attributes, but wouldn't be able to search them. So, this is to keep backwards compatibility until I remove standard visibility.